exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/StorageResources.java

```diff
@@ -221,6 +229,25 @@ private boolean isSupported(String format) {
             .collect(Collectors.toList());
   }
 
+  @POST
+  @Path("/storage/{name}")
+  @Consumes(MediaType.APPLICATION_JSON)
+  @Produces(MediaType.APPLICATION_JSON)
+  @Deprecated // use {@link #createOrUpdatePluginJSON} instead
```

Comment:
use `@Deprecated(".....")`

Reply:
This annotation hasn't description parameter.
I have added javadoc description